Fri Jun 03 00:19:14 NZST 2022 : Apache Derby Network Server - 10.14.2.0 - (1828579) started and ready to accept connections on port 1527
----------------------------------------------------------------
Fri Jun 03 00:19:15 NZST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-0181-245b-0906-0000334f5635 
on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\ChessDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/Maxth/Downloads/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=17.0.2+8-LTS-86
user.dir=C:\Users\Maxth\Downloads\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 03 00:22:12 NZST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 6965c036-0181-245b-0906-0000334f5635 
on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\testDb with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/Maxth/Downloads/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=17.0.2+8-LTS-86
user.dir=C:\Users\Maxth\Downloads\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 03 00:22:12 NZST 2022:
Shutting down instance 6965c036-0181-245b-0906-0000334f5635 on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\testDb with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Fri Jun 03 00:22:12 NZST 2022 Thread[DRDAConnThread_4,5,main] (DATABASE = testDb), (DRDAID = {3}), Database 'testDb' shutdown.
----------------------------------------------------------------
Fri Jun 03 00:22:15 NZST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance f67740d7-0181-245b-0906-0000334f5635 
on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\testDb with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/Maxth/Downloads/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=17.0.2+8-LTS-86
user.dir=C:\Users\Maxth\Downloads\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 03 00:28:54 NZST 2022:
Shutting down instance f67740d7-0181-245b-0906-0000334f5635 on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\testDb with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Fri Jun 03 00:28:54 NZST 2022 Thread[DRDAConnThread_5,5,main] (DATABASE = testDb), (DRDAID = {6}), Database 'testDb' shutdown.
----------------------------------------------------------------
Fri Jun 03 00:28:56 NZST 2022:
Shutting down instance a816c00e-0181-245b-0906-0000334f5635 on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\ChessDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Fri Jun 03 00:28:56 NZST 2022 Thread[DRDAConnThread_5,5,main] (DATABASE = ChessDB), (DRDAID = {7}), Database 'ChessDB' shutdown.
----------------------------------------------------------------
Fri Jun 03 00:29:05 NZST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance e66540fa-0181-245b-0906-0000334f5635 
on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\testDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/Maxth/Downloads/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=17.0.2+8-LTS-86
user.dir=C:\Users\Maxth\Downloads\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 03 00:29:06 NZST 2022:
Shutting down instance e66540fa-0181-245b-0906-0000334f5635 on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\testDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Fri Jun 03 00:29:06 NZST 2022 Thread[DRDAConnThread_5,5,main] (DATABASE = testDB), (DRDAID = {9}), Database 'testDB' shutdown.
Fri Jun 03 00:29:59 NZST 2022 Thread[DRDAConnThread_5,5,main] Cleanup action starting
java.sql.SQLException: Database 'testDB' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: ERROR XJ004: Database 'testDB' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 21 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'testDB' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Fri Jun 03 00:29:59 NZST 2022 Thread[DRDAConnThread_5,5,main] (DATABASE = testDB), (DRDAID = {10}), Database 'testDB' not found.
----------------------------------------------------------------
Fri Jun 03 00:44:46 NZST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 7247c19b-0181-245b-0906-0000334f5635 
on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\ChessDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/Maxth/Downloads/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=17.0.2+8-LTS-86
user.dir=C:\Users\Maxth\Downloads\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Fri Jun 03 00:44:47 NZST 2022:
Shutting down instance 7247c19b-0181-245b-0906-0000334f5635 on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\ChessDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Fri Jun 03 00:44:47 NZST 2022 Thread[DRDAConnThread_3,5,main] (DATABASE = ChessDB), (DRDAID = {12}), Database 'ChessDB' shutdown.
----------------------------------------------------------------
Fri Jun 03 00:44:55 NZST 2022:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance b77a823c-0181-245b-0906-0000334f5635 
on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\ChessDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
Loaded from file:/C:/Users/Maxth/Downloads/db-derby-10.14.2.0-bin/db-derby-10.14.2.0-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=17.0.2+8-LTS-86
user.dir=C:\Users\Maxth\Downloads\db-derby-10.14.2.0-bin\db-derby-10.14.2.0-bin
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB
Database Class Loader started - derby.database.classpath=''
Fri Jun 03 02:18:56 NZST 2022 Thread[DRDAConnThread_5,5,main] (XID = 230), (SESSIONID = 2), (DATABASE = ChessDB), (DRDAID = {14}), Cleanup action starting
java.sql.SQLNonTransientConnectionException: Connection authentication failure occurred.  Reason: Invalid authentication..
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.checkUserCredentials(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: ERROR 08004: Connection authentication failure occurred.  Reason: Invalid authentication..
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 21 more
============= begin nested exception, level (1) ===========
ERROR 08004: Connection authentication failure occurred.  Reason: Invalid authentication..
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.checkUserCredentials(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Fri Jun 03 02:18:56 NZST 2022 Thread[DRDAConnThread_5,5,main] (DATABASE = ChessDB), (DRDAID = {14}), Connection authentication failure occurred.  Reason: Invalid authentication..
----------------------------------------------------------------
Fri Jun 03 03:00:07 NZST 2022: Shutting down Derby engine
----------------------------------------------------------------
Fri Jun 03 03:00:07 NZST 2022:
Shutting down instance b77a823c-0181-245b-0906-0000334f5635 on database directory C:\Users\Maxth\Documents\NetBeansProjects\P141_20125153_Assignment2\DB\ChessDB with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@73d16e93 
----------------------------------------------------------------
